src/components/Sidebar/SettingsPane.jsx

```diff
@@ -1,84 +1,152 @@
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
+import React from 'react';
+import { useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 
 import { SceneKey } from '../../api/keys';
 import {
   getLastWordOfUri, isDeadEnd, isPropertyOwnerHidden, isPropertyVisible
 } from '../../utils/propertyTreeHelpers';
 import { CaseInsensitiveSubstring, ListCaseInsensitiveSubstring } from '../../utils/StringMatchers';
-import subStateToProps from '../../utils/subStateToProps';
 import { FilterList, FilterListData, FilterListFavorites } from '../common/FilterList/FilterList';
 import LoadingBlocks from '../common/LoadingBlock/LoadingBlocks';
 
 import Pane from './Pane';
 import SettingsPaneListItem from './SettingsPaneListItem';
 
-class SettingsPane extends Component {
-  render() {
-    const defaultEntries = this.props.topPropertyOwners.map((p) => ({
+function collectOwnersRecursively(propertyOwners, owners, allOwners) {
+  owners.forEach((uri) => {
+    const subowners = propertyOwners[uri]?.subowners || [];
+    // Parameter assignment necessary for recursion
+    // eslint-disable-next-line no-param-reassign
+    allOwners = allOwners.concat(subowners);
+    // eslint-disable-next-line no-param-reassign
+    allOwners = collectOwnersRecursively(propertyOwners, subowners, allOwners);
+  });
+
+  // Stops when owners are empty
+  return allOwners;
+}
+
+function collectPropertiesRecursively(propertyOwners, owners, allProperties) {
+  owners.forEach((uri) => {
+    const subowners = propertyOwners[uri]?.subowners || [];
+    const propertiesUri = propertyOwners[uri]?.properties || [];
+    // Parameter assignment necessary for recursion
+    // eslint-disable-next-line no-param-reassign
+    allProperties = allProperties.concat(propertiesUri);
+    // eslint-disable-next-line no-param-reassign
+    allProperties = collectPropertiesRecursively(propertyOwners, subowners, allProperties);
+  });
+
+  // Stops when owners are empty
+  return allProperties;
+}
+
+function SettingsPane({
```

Comment:
How about: 
```
function SettingsPane({ closeCallback }) {
```
since there is only one property? What does the linting think about that?